app/views/albums/index.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="albums__topNav">
+    <%= button_to 'ADD NEW', new_album_path, { method: 'get', class: "albums__addButton" } %>
+</div>
+
+<table class="albums__table">
+  <tr class="albums__tableHeader">
+    <th><h3>Title</h3></th>
+    <th><h3>Artist</h3></th>
+    <th><h3>Details</h3></th>
+    <th><h3>Delete</h3></th>
+  </tr>
+  <% if @albums.length > 0 %>
+    <% @albums.each_with_index do |album, i| %>
+      <tr class=
+        <% if i % 2 == 0 %>
+            <%= "albums__tableRow" %>
+          <% else %>
+            <%= "albums__tableRowAlt" %>
+          <% end %>
+        >
+        <th>
+          <%= album.title %>
+        </th>
+        <th>
+          <%= album.artist %>
+        </th>
+        <th>
+          <%= button_to 'MORE >>', album_path(album.id), method: 'get' %>
```

Comment:
I believe the `button_to` helper can figure out the path on its own if you just did
```erb
<%= button_to 'MORE >>', album, method: 'get' %>
```
both should be valid